core/arch/arm/plat-imx/imx_caam.h

```diff
@@ -32,13 +33,53 @@ struct imx_caam_job_ring {
 #define CCM_CCGR0_CAAM_WRAPPER_ACLK	SHIFT_U32(3, 10)
 #define CCM_CCGR6_EMI_SLOW		SHIFT_U32(3, 10)
 
+/* jr configuration registers */
+struct imx_caam_jr_ctrl {
+	uint32_t	irbar;
+	uint32_t	irbar_ls;
+	uint8_t		padding[4];
+	uint32_t	irsr;
+	uint8_t		padding2[4];
+	uint32_t	irsar;
+	uint8_t		padding3[4];
+	uint32_t	irjar;
+	uint32_t	orbar;
+	uint32_t	orbar_ls;
+	uint8_t		padding4[4];
+	uint32_t	orsr;
+	uint8_t		padding5[4];
+	uint32_t	orjrr;
+	uint8_t		padding6[4];
+	uint32_t	orsfr;
+	uint8_t		padding7[4];
+	uint32_t	jrstar;
+	uint8_t		padding8[4];
+	uint32_t	jrintr;
+	uint32_t	jrcfgr_ms;
+	uint32_t	jrcfgr_ls;
+	uint8_t		padding9[4];
+	uint32_t	irrir;
+	uint8_t		padding10[4];
+	uint32_t	orwir;
+	uint8_t		padding11[4];
+	uint32_t	jrcr;
+	uint8_t		padding12[1688];
+	uint32_t	jraav;
+	uint8_t		padding13[2296];
```

Comment:
Cross check the structure. I could 4100 bytes, i would guess it is 4096 bytes aligned.